src/index.ts

```diff
@@ -15,11 +15,12 @@
  * This also includes utility functions for operating on GraphQL types and
  * GraphQL documents to facilitate building tools.
  *
- * You may also import from each sub-directory directly. For example, the
- * following two import statements are equivalent:
- *
+ * You may also import from each sub-directory directly.
+ * @example the following two import statements are equivalent:
```

Comment:
Let's decide on `@example` latter, so for now just convert the code sample into markdown code blocks